Add 2021 CEPCI Value

bpaul4 opened the pull request:
## Fixes


## Summary/Motivation:


## Changes proposed in this PR:
- Adding `USD_2021` units to the IDAES currency dictionary, as these units will be required for applications in other subtasks.

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

andrewlee94 reviewed (changes requested):
1. Just to check, can you provide a reference for this value?
2. You need to update the tests to cover this new entry.

bpaul4 commented:
1. I double checked the source, and updated it accordingly to use the full year aggregate and not just a partial year estimate. I added the source to the docstring.
2. I added the new entry to the test file.
